chapter_07_aggregate.asciidoc

```diff
@@ -847,12 +849,11 @@ https://www.2ndquadrant.com/en/blog/postgresql-anti-patterns-read-modify-write-c
 
 //TODO maybe better diagrams here?
 
-Again, in our simple case, it's not obvious which to prefer.  In a more complex
-scenario, `SELECT FOR UPDATE` might lead to more deadlocks, while `REPEATABLE
-READ` would lead to more retries.  But if you have a test like the
-one we've shown, you can specify the behavior you want, and see how
-it changes.  You can also use the test as a basis for performing
-some performance experiments.
+We don't really have time to discuss all the tradeoffs between `REPEATABLE READ`
```

Comment:
To clarify - I'm not sure it's right to be comparing an isolation level with a SQL command...that doesn't make sense to me.

Reply:
If you add an example of another DB's OCC mechanism (e.g., [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/optimistic-concurrency-control.html)), it may help clarify the point and make a clean segue between this section and the next.